### DISCUSSION TITLE
Question about the API

### DISCUSSION THREAD

**Discussion by iced-queen** [GitbookIO/community#178] [0. General bugs and questions] [answered]
### Ask a question, start a conversation, or make an annoucement

Hey, I have a question about the `spaces/{spaceId}/search/ask` endpoint.
Is there a way to tell if it did not find an answer to a question?
So I can handle answers like "I don't know"

**Comment by SamyPesse** [comment, marked as answer]
The API will not return the `answer` object if we can't find an answer to a question.

**Reply by iced-queen** [reply]
Sometimes I get answers like `That is not a question. It seems to be a statement. Can you please provide a question?`
